cmd/utils/nodecmd/util.go

```diff
@@ -0,0 +1,220 @@
+package nodecmd
```

Comment:
@hyunsooda Looks like we are missing a copyright notice here.

Reply:
Thanks added.